app/controllers/orga/conferences_controller.rb

```diff
@@ -1,53 +1,58 @@
 class Orga::ConferencesController < Orga::BaseController
+  before_action :find_resource, only: [:show, :edit, :update, :destroy]
 
+  def index
+    @conferences = conferences
+  end
+  
+  def new
+    @conference = Conference.new
+  end
+  
   def create
-    conference.season = current_season
-    respond_to do |format|
-      if conference.save
-        format.html { redirect_to orga_conference_path(conference) }
-      else
-        format.html { render action: :new }
-      end
+    @conference = Conference.new(conference_params.merge(season: current_season))
+    if @conference.save
+     redirect_to orga_conference_path(@conference)
+    else
+      render action: :new
     end
   end
-
+  
   def update
-    if conference.update(conference_params)
-      redirect_to orga_conference_path(conference)
+    if @conference.update(conference_params)
+      redirect_to orga_conference_path(@conference)
     else
       render action: :edit
     end
   end
 
   def destroy
-    conference.destroy!
+    @conference.destroy!
     redirect_to orga_conferences_path, notice: 'The conference has been deleted.'
   end
 
   private
 
-  def conferences
-    @conferences ||= Conference.ordered(sort_params).in_current_season
+  def find_resource
+    @conference ||= Conference.find(params[:id])
```

Comment:
Yes we do, but not in that *"clever"* way it was used before with utilizing memorization to spare some lines in the controller code. Now, with doing it more or less explicit (I'm not the hugest fan of before filters...) we can spare the memorization part - the helper method will always just be called once.